test/disability-benefits/526EZ/config/vaFacilities.unit.spec.jsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import { mount } from 'enzyme';
+
+import { DefinitionTester, fillData } from '../../../util/schemaform-utils.jsx';
+import formConfig from '../../../../src/js/disability-benefits/526EZ/config/form.js';
+
+const initialData = {
```

Comment:
What do you think about pulling this `initialData` out into a separate file? We could then use it in all these test _and_ as the initial data in the `form.js` (until we're done building the form, of course).

Reply:
Good idea, updated.